test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -153,8 +158,50 @@ private enum RemoveType
 
     private Command<State<S>, Void, ?> waitForCMSToQuiesce()
     {
-        return new SimpleCommand<>(state -> "Waiting for CMS to Quiesce" + state.commandNamePostfix(),
-                                   state -> ClusterUtils.waitForCMSToQuiesce(state.cluster, state.cmsGroup));
+        return new Property.StateOnlyCommand<>()
+        {
+            private Epoch maxEpoch = null;
+            @Override
+            public String detailed(State<S> state)
+            {
+                if (maxEpoch == null)
```

Comment:
rather than guessing what epoch we wait on, add it to the history log... this means I need to *defer* finding the epoch until ready to execute...

this is called as a chain of `addNode, waitForCMSToQuiesce` so if we get the epoch right away its *before* the `addNode`, but we need the epoch *after* `addNode`